Tools/TestWebKitAPI/Tests/WebCore/gstreamer/GStreamerTest.cpp

```diff
@@ -66,6 +66,43 @@ TEST_F(GStreamerTest, gstStructureJSONSerializing)
     ASSERT_EQ(jsonString, "{\"words\":[\"hello\",\"world\"]}"_s);
 }
 
+TEST_F(GStreamerTest, codecStringParsing)
+{
+    using namespace GStreamerCodecUtilities;
+
+    ASSERT_STREQ(parseHEVCProfile("hev1.1.6.L93.B0"_s), "main");
+    ASSERT_STREQ(parseHEVCProfile("hev1.2.4.L93.B0"_s), "main-10");
+    ASSERT_STREQ(parseHEVCProfile("hev1.3.E.L93.B0"_s), "main-still-picture");
+    ASSERT_STREQ(parseHEVCProfile("hev1.4.16.L186.BF.C8"_s), "monochrome");
```

Comment:
Per `ISO-14496-15Y2019 E3`:

```
the 32 bits of the general_profile_compatibility_flags, but in reverse bit order, i.e. with general_profile_compatibility_flag[ 31 ] as the most significant bit, followed by general_ profile_compatibility_flag[ 30 ], and down to general_profile_compatibility_flag[ 0 ] as the least significant bit, where general_profile_compatibility_flag[ i ] for i in the range of 0 to 31, inclusive, are specified in ISO/IEC 23008-2, encoded in hexadecimal (leading zeroes may be omitted)
```

Thus `general_profile_compatibility_flags` should be encoded in `hexadecimal` not `decimal`, so I guess you need to replace this to `hev1.4.10.L186.BF.C8`